Firestore/core/src/firebase/firestore/core/sync_engine.h

```diff
@@ -47,6 +46,44 @@ namespace firebase {
 namespace firestore {
 namespace core {
 
+/**
```

Comment:
What's the reason to rename this from `SyncEngineCallback`?

Reply:
Reasonable..I'll just name it back.